### DISCUSSION TITLE
Dinosaur Trek

### DISCUSSION THREAD

**Discussion by eddiecarbin** [google/model-viewer#2821] [Show and tell]
Thank you for such awesome tech.  Model-viewer, provided me the ability to add WebAR to this web app created for Jurupa Mountains Discovery Center in Riverside, CA.   It's a scavenger hunt game like Pokeman Go inside the museum.   Discover your first dinosaur, dig for fossils, and collect extinct animals that once roamed the earth. 

https://sites.google.com/view/dinosaursafari-jmdc/home

Pro Tip: Press the setting button ten times fast to unlock all the dinosaurs.

![Brachiosaurus@0 5x](https://user-images.githubusercontent.com/1911872/134944301-65add0d5-8299-4658-91fb-0bfe8972b974.png)

![Dimetrodon@0 5x](https://user-images.githubusercontent.com/1911872/134944348-74d65799-fe43-4672-a466-a73156166784.png)

![Trex@0 5x](https://user-images.githubusercontent.com/1911872/134944387-b5ef7238-e1a4-446c-840f-de08cfb8b54b.png)

**Comment by elalish** [comment]
Very cool, thanks for sharing!

**Comment by elalish** [comment]
tweeted: https://twitter.com/modelviewer/status/1442525877464539144

**Comment by YhonLimon** [comment]
Amazing work!
